tests/unittests/Foundation/NSFileManagerTests.mm

```diff
@@ -219,7 +219,7 @@
 
     NSDictionary* attributes = @{
         NSFileType : NSFileTypeRegular,
-        NSFileImmutable : @YES,
```

Comment:
You should set it to immutable, and then set it to mutable... or test a different flag entirely.